go.mod

```diff
@@ -5,7 +5,6 @@ go 1.15
 require (
 	github.com/dave/jennifer v1.3.0
 	github.com/ethereum/go-ethereum v1.9.22
-	github.com/hashicorp/golang-lru v0.5.4
```

Comment:
A little suspicious of modifying imports here, but I couldn't run the tests without adding Clickhouse. Then running `go mod tidy` fixed that up, but also got rid of this 🤔 

Figure maybe we bite the bullet now and restore this import if it turns out it's somehow necessary for plugin compatibility (and/or fix the issue downstream)

Reply:
Go for it. If there's a plugin compatibility problem that should really be addressed at the plugin.